client/src/components/UpcomingInterviewTable.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useEffect, useContext } from 'react';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import { useStyles } from '../themes/theme';
+import { getStandardTime } from '../utils/timeFunctions'
+import { CustomButton } from './Buttons'
+import { useHistory } from 'react-router';
+import { cancelInterview } from '../utils/apiFunctions';
+import { fetchInterviews } from '../utils/fetchInterviews';
+import { store } from '../context/store';
+import Typography from '@material-ui/core/Typography';
```

Comment:
We can combine all of the material ui imports from /core to make this a bit cleaner